tests/filtertransformers/test_elasticsearch.py

```diff
@@ -62,7 +62,6 @@ def transformer():
     ("nelements > 1 OR elements LENGTH = 1 AND nelements = 2", 4),
     ("(nelements > 1 OR elements LENGTH = 1) AND nelements = 2", 3),
     ("NOT elements LENGTH = 1", 3),
-    ("_exmpl2_field = 2", 1),
```

Comment:
Why do you skip this test? It is the only one with a prefix, so it seems to test something that is not in the other tests.

Reply:
Can't remember why I removed this, might have been by accident. Let me test with it enabled again.